bus/src/main/java/org/hawkular/metrics/component/publish/AvailDataMessage.java

```diff
@@ -104,6 +104,8 @@ public String toString() {
         @JsonInclude
         private String tenantId;
         @JsonInclude
+        private String type;
```

Comment:
I don't think we should add this, it's an unlikely future and can cause (actually has caused) confusion.